opentuner/search/technique.py

```diff
@@ -223,17 +247,18 @@ def all_techniques(args):
     if m:
       import_module('opentuner.search.'+m.group(1))
 
-  return the_registry
+  return the_registry, the_composable_registry
 
 
 def get_enabled(args):
-  techniques = all_techniques(args)
+  techniques, composables = all_techniques(args)
   if args.list_techniques:
     for t in techniques:
       print t.name
     sys.exit(0)
 
   if not args.technique:
+    # no techniques specified, default technique
     args.technique = ['AUCBanditMetaTechniqueA']
```

Comment:
I'm wondering if Kevin has run any experiments that would give us a better guess at this point.  100% agreed we should make an evidence based choice.